SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -989,6 +989,43 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 	// SpatialGDK - Here Unreal would return the position of the last replicated actor in PriorityActors before the channel became saturated.
 	// In Spatial we use ActorReplicationRateLimit and EntityCreationRateLimit to limit replication so this return value is not relevant.
 }
+
+void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction* Function, void* Parameters)
+{
+	// The RPC might have been called by an actor directly, or by a subobject on that actor
+	UObject* CallingObject = SubObject != nullptr ? SubObject : Actor;
+
+	if (IsServer())
+	{
+		// Creating channel to ensure that object will be resolvable
+		GetOrCreateSpatialActorChannel(CallingObject);
+	}
+
+	int ReliableRPCIndex = 0;
+	if (GetDefault<USpatialGDKSettings>()->bCheckRPCOrder)
+	{
+		if (Function->HasAnyFunctionFlags(FUNC_NetReliable) && !Function->HasAnyFunctionFlags(FUNC_NetMulticast))
+		{
+			ReliableRPCIndex = GetNextReliableRPCId(Actor, FunctionFlagsToRPCSchemaType(Function->FunctionFlags), CallingObject);
+		}
+	}
+
+	TSet<TWeakObjectPtr<const UObject>> UnresolvedObjects;
+	RPCPayload Payload = Sender->CreateRPCPayloadFromParams(CallingObject, Function, ReliableRPCIndex, Parameters, UnresolvedObjects);
+
+	if (UnresolvedObjects.Num() == 0)
+	{
+		FPendingRPCParamsPtr RPCParams = MakeShared<FPendingRPCParams>(CallingObject, Function, MoveTemp(Payload), ReliableRPCIndex);
+		Sender->QueueOutgoingRPC(RPCParams);
+		// TODO(Alex): Create jira ticket to remove queueing when not needed
```

Comment:
A todo for making a jira ticket seems excessive :P

Reply:
Handle this TODO before this goes in and put the ticket number here.

Reply:
Fixed it in this PR.